python/mxnet/ndarray/ndarray.py

```diff
@@ -869,7 +869,7 @@ def _sync_copyfrom(self, source_array):
         source_array = np.ascontiguousarray(source_array, dtype=self.dtype)
         if source_array.shape != self.shape:
             raise ValueError('Shape inconsistent: expected %s vs got %s'%(
-                str(self.shape), str(source_array.shape)))
+                str(source_array.shape), str(self.shape)))
```

Comment:
why switch the order of expected vs got?

Reply:
@zhreshold could you confirm if the above issue is right or not? coz the value error it gave sounded confusing due to mismatched shapes.